src/hooks/useSelectedNetwork.ts

```diff
@@ -35,5 +36,9 @@ export const useSelectedNetwork = (): {
     [dispatch, disconnect, setLocalStorageNetwork]
   );
 
-  return { switchNetwork, selectedNetwork };
+  return {
+    switchNetwork,
+    selectedNetwork,
+    selectedDydxChainId: ENVIRONMENT_CONFIG_MAP[selectedNetwork as DydxNetwork].dydxChainId as DydxChainId,
+  };
```

Comment:
replace with `getSelectedDydxChainId` in `appSelectors`

Reply:
or we should only use one or the other. I don't see the need for a selector version and one from this hook.